### DISCUSSION TITLE
Why there's lack of resources in C++ for WinUi 3

### DISCUSSION THREAD

**Discussion by naughtykid001** [microsoft/WindowsAppSDK#4120] [General]
Is there any community or resources to pick up WinUi in C++?
Is it stable or comparable performance compare to C#?

**Comment by naughtykid001** [comment]
I'm actually looking for community to discuss and learn more about it. It does seems ambitious & exciting, but it just got to be stable in reality.

**Comment by jaigak** [comment]
> Is it stable or comparable performance compare to C#?

Performance is actually much better than in C# as the interop is slower in dotnet.
